recon/src/libp2p.rs

```diff
@@ -2,12 +2,12 @@
 //!
 //! There are various types within this module and its children.
 //!
-//! Behaviour - Responsible for coordiating the many concurrent Recon syncs
+//! Behavior - Responsible for coordinating the many concurrent Recon syncs
```

Comment:
nit, the libp2p crate uses the British spelling `Behaviour` while less common we should use it for consistency. Also this comment referes to a type name `Behaviour` so the spelling is important. I should have used the link syntax to make that clear. I'll do that in another PR.